### DISCUSSION TITLE
Jitsi Meet with ejabberd viable in 2024? Who is running this combo successfully?

### DISCUSSION THREAD

**Discussion by jeroenhabets** [processone/ejabberd#4239] [Questions] [answered]
Hi,

I can only find old information on running Jitsi Meet with ejabberd...

Some blog posts, like:
https://blog.windfluechter.net/2020/06/19/1759-jitsi-meet-and-ejabberd/
and comments like https://github.com/processone/ejabberd/discussions/3779#discussioncomment-2295771

And I noticed @weiss, @licaon-kter actively pursuing support a few years back (both here and at jitsi), e.g.
https://github.com/processone/ejabberd/issues/1861#issuecomment-315038626
https://github.com/jitsi/jitsi-meet/issues/5786

This could mean that Jitsi + ejabberd has been running fine for some people (perhaps with some limitations accepted for their usage). Or people like @jirkanov have given up and lost interest... 

Hence my questions in the subject that's hopefully good info to have available for others:

Is "Jitsi Meet with ejabberd viable in 2024? Who is running this combo successfully?"

If you are, any limitations/risks/regrets I need to be aware of?
If you've given up, that's good to know about too.

TIA!

**Comment by licaon-kter** [comment]
There was this wave back in 2020, but more apps added at least 1:1 calls (and the whole _reason_ for remote work went away ahem)

In my case I just given up, it was a pain to even build for arm64.

Jitsi Meet didn't care about being part of the XMPP community, having the startup mentality, pushing for more custom LUA modules and REST APIs, not XMPP interoperability.

If I'd deploy now I'd just setup a prosody along side. But I'm interested to know too, if that's not needed.

**Comment by weiss** [comment, marked as answer]
> In my case I just given up

Same here.  For some newer Jitsi Meet features, you need non-standard Prosody plugins written by the Jitsi people, and I have no plans to port those to ejabberd.

**Reply by jeroenhabets** [reply]
Thank you both for confirming my hunch. And as far as I can find, Jitsi Meet is the only XMPP group call solution so I need to weigh if I need to keep that option open...

For anyone find this later, please do chime in especially you are (interested in|happily|abandoned) running this combo.

**Reply by mremond** [reply]
We plan to work to provide group calls directly with ejabberd and possibly an external component for performance. Architecture is not fully defined yet.

**Reply by licaon-kter** [reply]
@mremond you are building on the work that Dino devs did (afaik) for their group calls?

**Reply by mremond** [reply]
Yes, we are studying what has been done already. We would like to be as standard and operable as possible between client implementations.

**Comment by oliviasculley** [comment]
I'm still trying to get this to work but stuck at getting the components to work in ejabberd. Here's some resources that I found that were helpful:
- https://blog.jabberhead.tk/2020/03/16/install-jitsi-meet-alongside-ejabberd/
- https://github.com/jitsi/jitsi-meet/issues/1306
- https://mmujtabaroohani.medium.com/why-i-failed-to-install-jitsi-meet-alongside-ejabberd-12d903d9116e
- https://community.jitsi.org/t/focus-component-logic/100540
